Create stakezone.json

stakezone opened the pull request.

stakezone committed:
Create stakezone.json

stakezone committed:
Update stakezone.json

jackzampolin commented:
For some reason I'm unable to create a lite client for your chain. I'm not sure why, so I'm going to investigate soon. Going to hold off merge.

stakezone committed:
amended chain_id

stakezonechain as chain_id does not work for the setup procedure, it gets amended into stakezonezonechain in genesis.json. Apparently 'zone' is somehow a reserved keyword.

stakezone commented:
Problems were caused by a wrong chainid in genesis.json. Chosing a chainid name that does not contain the string 'zone' resolved the issue.

jackzampolin commented:
Closing in favor of the https://github.com/cosmosdevs/GameOfZones repo.
